doc/release-notes/5.6-release-notes.md

```diff
@@ -0,0 +1,120 @@
+
+### Guestbook Responses API
+
+A new API to retrieve Guestbook responses has been added. This makes it easier to retrieve the records for large guestbooks and also makes it easier to integrate with external systems.
+
+### Dataset Semantic API (Experimental)
+
+Dataset metadata can be retrieved, set, and updated using a new, flatter JSON-LD format - following the format of an OAI-ORE export (RDA-conformant Bags), allowing for easier transfer of metadata to/from other systems (i.e. without needing to know Dataverse's metadata block and field storage architecture). This new API also allows for the update of terms metadata (#5899).
+
+This development was supported by the [Research Data Alliance](https://rd-alliance.org), DANS, and Sciences PO and follows the recommendations from the [Research Data Repository Interoperability Working Group](http://dx.doi.org/10.15497/RDA00025).
+
+### Direct Upload API Now Available for Adding Multiple Files Metadata to the Dataset
+
+Users can now add metadata of multiple files to the dataset once the files exists in the s3 bucket using the direct upload API. This makes direct uploads more efficient and reduces server load by only updating the dataset once instead of once per file. For more information, see the [Direct DataFile Upload/Replace API section](https://guides.dataverse.org/en/5.6/developers/s3-direct-upload-api.html) of the Dataverse Software Guides.
+
+## Major Use Cases
+
+Newly-supported major use cases in this release include:
+
+- Users can create Private URLs that anonymize dataset metadata, allowing for double blind peer review. (Issue #1724, PR #7908)
+- Users can download Guestbook records using a new API. (Issue #7767, PR #7931)
+- Users can update terms metadata using the new semantic API. (Issue #5899, PR #7414)
+- Users can retrieve, set, and update metadata using a new, flatter JSON-LD format. (Issue #6497, PR #7414)
+- Administrators can use the Traces API to retreive information about specific types of user activity  (Issue #7952, PR #7953)
+
+## Notes for Dataverse Installation Administrators
+
+### New Database Constraint
+
+A new DB Constraint has been added in this release. Full instructions on how to identify whether or not your database needs any cleanup before the upgrade can be found in the [Dataverse software GitHub repository](https://github.com/IQSS/dataverse/blob/develop/scripts/issues/7451/PRE-RELEASE-INFO.txt). This information was also emailed out to Dataverse installation contacts.
+
+### Payara 5.2021.4 (or Higher) Required
+
+Some changes in this release require an upgrade to Payara 5.2021.4 or higher.
+
+Instructions on how to update can be found in the [Payara documentation](https://docs.payara.fish/community/docs/5.2021.4/documentation/user-guides/upgrade-payara.html) We've included the necessary steps below, but we recommend that you review the Payara upgrade instructions as it could be helpful during any troubleshooting.
```

Comment:
> We've included the necessary steps below ... 

Can we move this entire paragraph above to the upgrade instructions section? 
Even I was confused on the first reading of it... 
All we have in the upgrade instructions related to the payara upgrade is this line: 
> Unzip the new Payara version in its place

I really think we should always assume that the person reading the upgrade section has jumped there directly, without reading the entire note. 
OK if I try to rearrange this?

Reply:
@landreev - feel free to rearrange, thanks!

Reply:
OK, done. Feel free to refine/correct further. (overall, I approve!)